.gitignore

```diff
@@ -1 +1,4 @@
-molecule/default/molecule.yml
+# having the molecule.yml in .gitignore prevents the molecule converge to run.
```

Comment:
Do we have any risk of committing this from our envs? Could we avoid this somehow?

Reply:
yeap is not so good, but i didn't find a way to workaround this (didn't spent too much time tbh). I see people complaining about the same [here](https://github.com/ansible/molecule/issues/4117)